shared/src/test/scala/pl/metastack/metatime/TimeTest.scala

```diff
@@ -23,6 +23,13 @@ object TimeTest extends SimpleTestSuite with TestIgnored {
     assertEquals(Time(23, 10) > Hour(22), true)
     assertEquals(5.hours - 1.minute, 4.hours + 59.minutes)
   }
+  
+  test("plusMinusTime")
+  {
+    assertEquals((Time((Hour(30) + Hour(20)).asInstanceOf[Time])).equals(Time( hour=50)), true)
+	  assertEquals((Time((Minute(20) + Second(40)).asInstanceOf[Time])).equals(Time( hour=10, minute=20)), false)
```

Comment:
Don't put the curly brace on a new line. The indentation is off in line 30. The casts should not be necessary.